pandas/core/generic.py

```diff
@@ -8180,7 +8180,7 @@ def interpolate(
                         stacklevel=find_stack_level(),
                     )
 
-        if "fill_value" in kwargs:
+        if method in fillna_methods and "fill_value" in kwargs:
```

Comment:
@phofl Would it make sense to adapt the Error message below to mention that fill_value is not valid for the fillna-methods?

Reply:
Yeah I updated the message